src/brevitas/graph/gptq.py

```diff
@@ -204,7 +262,7 @@ def update_batch(self, module, input, out, current_layer):
             # Preprocess input by group
             for i, inp in enumerate(inp_by_group):
                 inp = unfold(inp)
-                inp = inp.permute([1, 0, 2])
+                inp = inp.permute([1, 0])
```

Comment:
What happened here?

Reply:
We are just interested in permuting the first two dimensions, leaving all the others unchanged. The `2` was redundant.